code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/InAppMessagePresentable.kt

```diff
@@ -108,9 +109,9 @@ internal class InAppMessagePresentable(
                         // So always dismiss the message when a gesture is detected
                         dismiss()
 
-                        // If a gesture mapping exists, the notify the listener about the url associated with the gesture
+                        // If a gesture mapping exists, the notify the listener about the uri associated with the gesture
                         inAppMessage.settings.gestureMap[gesture]?.let { link ->
-                            handleInAppUrl(link)
+                            handleInAppUri(link)
```

Comment:
We do have instrumentation tests that cover part of this logic partially. However, we will need to enhance them if possible to identify if the URI is loaded after the interaction.